projects/spectator/src/lib/base/dom-spectator.ts

```diff
@@ -123,7 +123,7 @@ export abstract class DomSpectator<I> extends BaseSpectator {
     this.detectChanges();
   }
 
-  public click(selector: SpectatorElement = this.element): void {
+  public click(selector: SpectatorElement | DOMSelector = this.element): void {
```

Comment:
You can just add it to the `SpectatorElement` type.

Reply:
Thanks, awesome advice that make fewer changes in code